examples/prometheus_exporter/README.md

```diff
@@ -27,22 +27,24 @@ docker run -d --expose 8080 \
 ```
 
 ## Run Example Program
+
+To run the program, you must provide a command line argument to specify which Aggregator Kind you want to run the demo. The six choices are:
+```
+counter, gauge, mmsc, histogram, exact, sketch
+```
+
 Run with Bazel:
 ```shell script
 bazel build //examples/prometheus_exporter:prometheus_exporter_example
-bazel-bin/examples/prometheus_exporter/prometheus_exporter_example
+bazel-bin/examples/prometheus_exporter/prometheus_exporter_example agg 
```

Comment:
agg -> counter so people can copy-paste this example.
 
"agg" is like a usage statement, "counter" is an example, which is what this block is :)

Reply:
Good catch, changed "agg" to "histogram" instead of "counter" to be more consistent with the example